src/MARIGOLD/iate.py

```diff
@@ -0,0 +1,364 @@
+    Dh              = cond.Dh                                   # Hydraulic diameter
+    LoverD          = cond.LoverD                               # Condition L/D
+
+    # Yadav
+    RoverD = 9      # Make into optional input
+    R_curv          = RoverD * Dh                               # Radius of curvature of elbow/bend
+
+    # May want to include arc of the bend?
+    # Ah, yes
+    L_elb = np.pi/2 * R_curv        # This assumes 90\degree elbow
+    # So, Yadav defines under Expcond.m the length of the vertical section -- I think this is how he toggles between VU, H, VD, elbow
+    # Instead, we probably would want to have an input switch indicating which mode of IATE we're executing
+
+    ############################################################################################################################
+    #                                                                                                                          #
+    #                                                       CONSTANTS                                                          #
+    #                                                                                                                          #
+    ############################################################################################################################
+    # expcond.m
+
+    p_atm           = 101325                                    # Ambient pressure [Pa]
+    rho_f           = 998.0                                     # Liquid phase density [kg/m**3]
+    rho_g           = 1.204                                     # Gas phase density [kg/m**3]
+    mu_f            = 0.001002                                  # Viscosity of water [Pa-s]
+    sigma           = 0.0728                                    # Surface tension of air/water [N/m]
+    grav            = 9.81*np.sin((theta)*np.pi/180)            # Gravity constant (added by Drew to account for pipe inclination)
+
+    # Worosz
+    R_spec          = 287.058                                   # Specific gas constant for dry air [J/kg-K]
+    T               = 293.15                                    # Ambient absolute temperature [K], for calculating air density as a function of pressure along channel
+    
+    ############################################################################################################################
+    #                                                                                                                          #
+    #                                                      COEFFICIENTS                                                        #
+    #                                                                                                                          #
+    ############################################################################################################################
+    # coeff.m
+    
```

Comment:
Are these constants going to be user inputs? You could have them be set on a method basis or directly

Reply:
Resolved